### ISSUE TITLE
Consider generating colour blend and gradient samples

### ISSUE THREAD

**Issue by adrinux** [adrinux/palegen#7]
Dependent on issues #5, #6 and #2.

go-colorful has library functions to generate blends and gradients. It might be nice to have Palegen generate examples of those too but only behind a command line flag or arguments passed from a configuration file (hence the dependence on issue #2
